stresstest/src/test/java/io/crate/benchmark/LuceneDocCollectorBenchmark.java

```diff
@@ -112,6 +110,25 @@
             new RamAccountingContext("dummy", new NoopCircuitBreaker(CircuitBreaker.Name.FIELDDATA));
     private JobCollectContext jobCollectContext;
 
+
+    public class PausingCollectingRowReceiver extends CollectingRowReceiver {
```

Comment:
then i would have to update pauseAfter after every row, which would affect the benchmark results.